src/ansible/roles/common/tasks/main.yml

```diff
@@ -75,7 +75,7 @@
     state: absent
     remove: yes
     force: yes
-  when: regular_user_check.stdout | length > 0
+  when: regular_user_check.stdout | length > 0 and regular_user_check.stdout != user.regular.name
```

Comment:
Without checking I would be vary if there is not <LF> characters in  regular_user_check.stdout

Reply:
Works in my tests.